### PR TITLE
Add Welsh Translation 🏴󠁧󠁢󠁷󠁬󠁳󠁿

### PR DESCRIPTION
**Pull request by Stivaros**
🏴󠁧󠁢󠁷󠁬󠁳󠁿

* Adds bilingual support to ET3 for English and Welsh 🏴󠁧󠁢󠁷󠁬󠁳󠁿
* Adds Welsh strings 🏴󠁧󠁢󠁷󠁬󠁳󠁿
* Massive rewrite of the test suite to support i18n, namely for Welsh 🏴󠁧󠁢󠁷󠁬󠁳󠁿

### CODE REVIEW

File: app/services/et_api_handler.rb [hunk 1]
```diff
@@ -51,7 +51,9 @@ def self.build_response_data(full_hash)
         "make_employer_contract_claim": full_hash[:employer_contract_claim_answers][:make_employer_contract_claim],
         "claim_information": full_hash[:employer_contract_claim_answers][:claim_information],
         "additional_information_key": (full_hash[:additional_information_answers][:upload_additional_information] == "" ? nil : full_hash[:additional_information_answers][:upload_additional_information]),
-        "email_receipt": full_hash[:confirmation_of_supplied_details_answers][:email_receipt]
+        "email_receipt": full_hash[:confirmation_of_supplied_details_answers][:email_receipt],
+        "pdf_template_reference": "et3-v1-#{I18n.locale == :cy ? "cy" : "en"}",
```
Comment (garytaylor) [inline app/services/et_api_handler.rb]: I18n.locale is "en" or "cy" - no need for the ternary ?

File: app/views/layouts/application.html.slim [hunk 2]
```diff
@@ -1,3 +1,5 @@
+- content_for :html_lang
+  = params[:locale] || I18n.default_locale
```
Comment (garytaylor) [inline app/views/layouts/application.html.slim]: Why do this ?  you are already normalizing params[:locale] in the application controller and assigning the value to I18n.locale ?  So in this case, you could just use I18n.locale to get this value ?

File: config/routes.rb [hunk 3]
```diff
@@ -1,23 +1,25 @@
...
+      resource :employers_contract_claim, only: [:edit, :update], path_names: { edit: ''}
+      resource :additional_information, only: [:create, :edit, :update], path_names: { edit: ''}
+      resource :confirmation_of_supplied_details, only: [:edit, :update], path_names: { edit: ''}
+      get "form_submission", to: "form_submissions#index"
+    end
+    root 'static_pages#index'
+    get "/session_expired" => 'static_pages#expired'
+    get "/ping" => "status#ping"
+    get "/healthcheck" => "status#healthcheck"
+    delete "/respond/confirmation_of_supplied_details/remove_rtf" => 'confirmation_of_supplied_details#destroy_rtf', as: :remove_rtf
```
Comment (garytaylor) [inline config/routes.rb]: I assume you dont need a welsh version of this url ?

File: app/views/layouts/application.html.slim (hunk 2 above)
Comment (Stivaros) [reply app/views/layouts/application.html.slim]: Good point - fixed in 8d93eee

File: app/services/et_api_handler.rb (hunk 1 above)
Comment (Stivaros) [reply app/services/et_api_handler.rb]: Resolved in 53c6aaf thanks!

File: config/routes.rb (hunk 3 above)
Comment (Stivaros) [reply config/routes.rb]: Moved them out of the namespace here b1e39c1

**Pull request merged**